dev/tools/merge-pr.sh

```diff
@@ -15,12 +15,11 @@ API=https://api.github.com/repos/coq/coq
 
 BASE_BRANCH=`curl -s $API/pulls/$PR | jq -r '.base.label'`
 
-COMMIT=`git rev-parse $REMOTE/pr/$PR`
```

Comment:
This breaks the line coming just after...

Reply:
Fixed.